tests/functional/__init__.py

```diff
@@ -3,3 +3,31 @@
 #   This file is distributed under the University of Illinois Open Source
 #   License. See LICENSE.TXT for details.
 # -----------------------------------------------------------------------------
+import os
+import sys
+import json
+
+# setup the required environment variables for the tests
+
+__REPO_ROOT = os.path.abspath(os.environ['REPO_ROOT'])
+# sys.path modification needed so nosetests can load the libtest package.
+sys.path.append(os.path.abspath(__REPO_ROOT))

```

Comment:
The generated python modules are used from the built package which can not be found by the test running infrastructure.